### PR TITLE
Automatically retrieve ownerId from auth middleware when creating a project

### PR DESCRIPTION
**Pull request by Tom-Bonnike**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -17,6 +17,7 @@
       }
     ],
     "transform-es2015-modules-commonjs",
+    "transform-object-rest-spread",
```
Comment (Tom-Bonnike) [inline .babelrc]: Apparently this was missing, even with Node 9? 🤔

File: .env.dist [hunk 2]
```diff
@@ -14,5 +14,6 @@ POSTGRES_DATA_DIR=/var/lib/postgresql/data
 
 REDIS_DATA_DIR=/data
 REDIS_HOST=redis.host
+REDIS_PORT=6379
```
Comment (Tom-Bonnike) [inline .env.dist]: This was missing: https://github.com/ZolaApp/zola-api/pull/85/files#diff-321318d6aed05edfbf645099d1a622ffR11.

File: src/api/projects/mutations/createProject/index.js [hunk 3]
```diff
@@ -2,8 +2,23 @@
...
+  request: express$Request
+}
+
+const resolver = async (
+  _: any,
+  args: CreateProjectArgs,
+  { request }: Context
+) => {
+  if (request.user === null) {
+    throw new Error('Please log in.')
```
Comment (Tom-Bonnike) [inline src/api/projects/mutations/createProject/index.js]: I guess we could export an error from `@constants/error.js` to re-use the same in every resolver?

Comment (Tom-Bonnike) [reply src/api/projects/mutations/createProject/index.js]: I still think that ideally this should be done in one location and thrown if the operation requires to be authenticated 🤔

File: src/server/index.js [hunk 4]
```diff
@@ -26,7 +26,17 @@ export default (): express$Application => {
...
-  app.use(GRAPHQL_PATH, graphqlExpress({ schema }))
+  app.use(
+    GRAPHQL_PATH,
+    graphqlExpress(request => ({
+      schema,
+      context: {
+        // Pass down express’ `request` to the GraphQL context.
+        request
+      },
+      debug: process.env.node_ENV !== 'production'
```
Comment (Tom-Bonnike) [inline src/server/index.js]: I guess it’s nice to have that ☝️

**Comment by Tom-Bonnike** [comment]
I still need to fix tests.

File: src/api/projects/mutations/createProject/index.js (hunk 3 above)
Comment (PauuloG) [reply src/api/projects/mutations/createProject/index.js]: Like chaining a middleware specifically for GraphQL auth behind our global auth middleware ? How would you know which mutations/queries require auth and which don't ?

Comment (Tom-Bonnike) [reply src/api/projects/mutations/createProject/index.js]: I thought about it and I think we could totally have something like…
```js
const GRAPHQL_OPERATIONS_WITH_AUTH = ['createProject', …]
```

🤔 

And on the middleware, whitelist based on the operation name that I think we can retrieve?

File: .env.dist [hunk 5]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (Tom-Bonnike) [inline .env.dist]: oops

**Pull request merged**